zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -113,7 +110,7 @@ public Response postLogin(@FormParam("userName") String userName,
     return response.build();
   }
   
-  @POST
+  @GET
```

Comment:
Any reason to change logout from `POST` to `GET`? I dont think its aligned why HTTP standard.